codec-http/src/main/java/io/netty/handler/codec/http/HttpMethod.java

```diff
@@ -86,7 +87,7 @@
      */
     public static final HttpMethod CONNECT = new HttpMethod("CONNECT");
 
-    private static final Map<String, HttpMethod> methodMap = new HashMap<String, HttpMethod>();
+    private static final Map<String, HttpMethod> methodMap = new TreeMap<String, HttpMethod>();
```

Comment:
I have a few flight recorder results...however I think it warrants a closer look. Let me pull out into a another PR so we can look at this independently